pkg/metrics/metrics_test.go

```diff
@@ -45,17 +45,22 @@ func TestRegisterCustomMetrics(t *testing.T) {
 		assert.True(t, k8smetrics.Registry.Unregister(m))
 	}
 
+	for _, m := range allGaugeVecs {
+		assert.True(t, k8smetrics.Registry.Unregister(m))
+	}
 }
 
 func TestResetMetrics(t *testing.T) {
 
 	// when
 	UserSignupUniqueTotal.Inc()
 	MasterUserRecordGauge.Set(22)
+	MasterUserRecordGaugeVec.WithLabelValues("member-1").Set(20)
 
 	Reset()
 
 	// then
 	assert.Equal(t, float64(0), promtestutil.ToFloat64(UserSignupUniqueTotal))
 	assert.Equal(t, float64(0), promtestutil.ToFloat64(MasterUserRecordGauge))
+	assert.Equal(t, float64(0), promtestutil.ToFloat64(MasterUserRecordGaugeVec.WithLabelValues("member-1")))
```

Comment:
since every metric type has also its own unit test verifying the number that is being set: https://github.com/codeready-toolchain/host-operator/blob/53c13926ba73b3d631a9ac2d21bc4ec6ba8d13ac/pkg/metrics/metrics_test.go#L12-L32
could you please add one also for GaugeVec?

Reply:
done in 1475e93